js/plugins/lib/twoDigitFloatConverter.js

```diff
@@ -0,0 +1,13 @@
+export default function convertToFloatWith2Digit(number) {
+    if (!number) {
+        return 0;
+    }
+
+    const value = number.toString();
+
+    if (value.split('.')[1] && value.split('.')[1].length > 1) {
```

Comment:
@onur-dogan why do we need split operation here? There is no need to check decimals of numbers, `parseFloat(value).toFixed(n)` would be enough.

Reply:
I splitted value there because of if the number is integer like; 20 or if the number has no more than 1 number after dot. If we use toFixed on this value it returns as 20.00. 20.1 returns as 20.10. I thought that would be better to ignore these zero values after dot.